src/samplereader/FragmentedSampleReader.h

```diff
@@ -81,7 +81,7 @@ class ATTR_DLL_LOCAL CFragmentedSampleReader : public ISampleReader, public AP4_
   AP4_DataBuffer m_encrypted;
   AP4_DataBuffer m_sampleData;
   CodecHandler* m_codecHandler{nullptr};
-  const AP4_UI08* m_defaultKey{nullptr};
+  std::vector<uint8_t> m_defaultKey;
```

Comment:
Just checking - is this ok to now be uninitialised?

Reply:
before was nullptr so we have to do nothing more for vector
seem that we check for empty key here
https://github.com/xbmc/inputstream.adaptive/pull/1388/files#diff-c4b54d32a3ee2ca0e116448b56e9276218c27b93c81fce3abf402083263572abR894
at least for non android case